integration-tests/src/test/java/org/apache/fineract/integrationtests/loan/LoanApiIntegrationTest.java

```diff
@@ -0,0 +1,142 @@
+            PostLoanProductsResponse loanProductResponse2 = loanProductHelper.createLoanProduct(product2);
+            Long loanProductId2 = loanProductResponse2.getResourceId();
+
+            // Apply and Approve Loan
+            double amount = 5000.0;
+
+            PostLoansRequest applicationRequest = applyLoanRequest(clientId, loanProductId, "01 January 2023", amount, numberOfRepayments)//
+                    .repaymentEvery(repaymentEvery)//
+                    .interestRatePerPeriod(BigDecimal.valueOf(10.0))//
+                    .loanTermFrequency(numberOfRepayments)//
+                    .repaymentFrequencyType(RepaymentFrequencyType.MONTHS)//
+                    .loanTermFrequencyType(RepaymentFrequencyType.MONTHS)//
+                    .interestType(InterestType.DECLINING_BALANCE)//
+                    .interestCalculationPeriodType(InterestCalculationPeriodType.DAILY);//
+
+            PostLoansRequest applicationRequest2 = applyLoanRequest(clientId2, loanProductId2, "01 January 2023", amount,
+                    numberOfRepayments)//
+                    .repaymentEvery(repaymentEvery)//
+                    .interestRatePerPeriod(BigDecimal.valueOf(10.0))//
+                    .loanTermFrequency(numberOfRepayments)//
+                    .repaymentFrequencyType(RepaymentFrequencyType.MONTHS)//
+                    .loanTermFrequencyType(RepaymentFrequencyType.MONTHS)//
+                    .interestType(InterestType.DECLINING_BALANCE)//
+                    .interestCalculationPeriodType(InterestCalculationPeriodType.DAILY);//
+
+            PostLoansResponse postLoansResponse = loanTransactionHelper.applyLoan(applicationRequest);
+            PostLoansResponse postLoansResponse2 = loanTransactionHelper.applyLoan(applicationRequest2);
+
+            PostLoansLoanIdResponse approvedLoanResult = loanTransactionHelper.approveLoan(postLoansResponse.getResourceId(),
+                    approveLoanRequest(amount, "01 January 2023"));
+
+            PostLoansLoanIdResponse approvedLoanResult2 = loanTransactionHelper.approveLoan(postLoansResponse2.getResourceId(),
+                    approveLoanRequest(amount, "01 January 2023"));
+
+            Long loanId = approvedLoanResult.getLoanId();
+            Long loanId2 = approvedLoanResult2.getLoanId();
+            createdLoanId.getAndSet(loanId);
+            createdLoanId2.getAndSet(loanId2);
+
+            // disburse Loan
+            disburseLoan(loanId, BigDecimal.valueOf(amount), "01 January 2023");
+            disburseLoan(loanId2, BigDecimal.valueOf(amount), "01 January 2023");
+        });
+        runAt("01 February 2023", () -> {
+            long loanId = createdLoanId.get();
+            GetLoansLoanIdResponse loanResponse = loanTransactionHelper.getLoanDetails(loanId);
+            GetLoansResponse loansLoanIdResponse = loanTransactionHelper.retrieveAllLoans(loanResponse.getAccountNo(), clientId);
+            assert loansLoanIdResponse.getPageItems() != null;
```

Comment:
the language asserts won't do anything for you, they have to be enabled. If you wanna check against a null PageItems, use AssertJ.